### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by marcinpawlukiewicz**
Added commands: cd, touch, nano, cat, mkdir, mv, ls, pwd, rm, cp and additional information in a dropdown at the bottom.

### CODE REVIEW

File: resources/cheatsheet.md [hunk 1]
```diff
@@ -16,3 +16,65 @@ kernelspec:
...
+Remove a file:
+```
+rm file_name.ext
+```
+Copy a file:
+```
+cp file_name.ext copied_file_name.ext
+```
+
+``` {dropdown}
```
Comment (brownsarahm) [inline resources/cheatsheet.md]:
````suggestion
``` {note} 
````
I want to change this to a note box instead of a dropdown

File: resources/cheatsheet.md [hunk 2]
```diff
@@ -16,3 +16,65 @@ kernelspec:
...
+rm file_name.ext
+```
+Copy a file:
+```
+cp file_name.ext copied_file_name.ext
+```
+
+``` {dropdown} 
+1. In every command you can add your directory/ location (ex. docs/file_name.ext).
+2. "." dot symbolizes our current location and ".." two docts, a location one up.
```
Comment (brownsarahm) [inline resources/cheatsheet.md]:
```suggestion
2. "." dot symbolizes our current location and ".." two dots, one level up in the directory tree.
```

File: resources/cheatsheet.md [hunk 3]
```diff
@@ -16,3 +16,65 @@ kernelspec:
...
+3. "*" replaces all words after the symbol (ex. pyt*.py shows all files starting with word pyt and extension py).
```
Comment (brownsarahm) [inline resources/cheatsheet.md]: let's rephrase this to be a bit more clear. eg your example, it's not all the things after the `*` because you note that it would keep the `.py`

**Review by brownsarahm — CHANGES_REQUESTED** [review]
some minor fixes you can commit my suggestions and then one thing we should rephrase for clarity

File: resources/cheatsheet.md (hunk 3 above)
Comment (marcinpawlukiewicz) [reply resources/cheatsheet.md]: Done 👍
